File: linked_lists/remove_duplicates_from_sorted_list.java
class ListNode
{
    int val;
    ListNode next;
    ListNode (int val)
    {
        this.val = val;
    }
}

public ListNode deleteDuplicates(ListNode head)
{
    ListNode current = head;

    while (current != null && current.next != null)
    {
        if (current.val == current.next.val)
        {
            current.next = current.next.next;
        }
        else
        {
            current = current.next;
        }
    }

    return head;
}

File: linked_lists/remove_duplicates_from_sorted_list.py
class ListNode:
    def __init__(self, val: int):
        self.val = val
        self.next = None

def deleteDuplicates(head: Optional[ListNode]) -> Optional[ListNode]:
    current = head

    while current and current.next:
        if current.val == current.next.val:
            current.next = current.next.next
        else:
            current = current.next
    
    return head
